packages/office-ui-fabric-react/src/components/Slider/Slider.styles.ts

```diff
@@ -65,6 +74,15 @@ export const getStyles = (props: ISliderStyleProps): ISliderStyles => {
     }
   };
 
+  const slideBoxHoverThumbStyles = !disabled && {
+    border: `2px solid ${sliderHoverSectionColor}`,
+    selectors: {
+      [HighContrastSelector]: {
```

Comment:
nit: I see these HighContrast selector repeated several times. Maybe worth extracting into a const and reuse?